src/Explorer/Tabs/SettingsTab.ts

```diff
@@ -1617,17 +1564,15 @@ export default class SettingsTab extends TabsBase implements ViewModels.WaitsFor
       this.GEOMETRY;
     this.geospatialConfigType.setBaseline(geospatialConfigType);
 
-    if (!this.hasAutoPilotV2FeatureFlag()) {
-      const maxThroughput =
-        this.collection &&
-        this.collection.offer &&
-        this.collection.offer() &&
-        this.collection.offer().content &&
-        this.collection.offer().content.offerAutopilotSettings &&
-        this.collection.offer().content.offerAutopilotSettings.maxThroughput;
+    const maxThroughput =
+      this.collection &&
+      this.collection.offer &&
+      this.collection.offer() &&
+      this.collection.offer().content &&
+      this.collection.offer().content.offerAutopilotSettings &&
+      this.collection.offer().content.offerAutopilotSettings.maxThroughput;
```

Comment:
nit: probably a good place for conditional chaining to make it less verbose.

Reply:
I'll add a linting rule for this in another PR https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md